app/models/spree/content_item_decorator.rb

```diff
@@ -0,0 +1,9 @@
+module Spree
+  module Stock
+    ContentItem.class_eval do
+      def variant_has_product_packages?
```

Comment:
This can be done through a 
```
delegate :has_product_packages?, to: :variant, prefix: true
```
I am not sure that I see the value of adding this class eval though.